docker/elasticsearch/README.md

```diff
@@ -0,0 +1,5 @@
+#### 2.0 Start Elasticsearch with docker
+If you familier with docker you can use the elasticsearch docker image with installed elasticsearch-vector-scoring plugin.
+Just build the image with 
+```docker build -t es-with-vector-scoring .```
+and start the image with ```docker run -p 9200:9200 -p 9300:9300 es-with-vector-scoring```
```

Comment:
lastly, tell the user where to go back to, something like "Now that you've got Elasticsearch up and running, you can go back to step 3."